modules/labwhere.js

```diff
@@ -1,10 +1,9 @@
 import axios from 'axios'
 import config from '@/nuxt.config'
 
-const getPlatesFromBoxBarcodes = async (boxBarcodes) => {
+const getPlatesFromBoxBarcodes = async (...boxBarcodes) => {
   try {
-    const boxBarcodesParam = makeBoxBarcodesParam(boxBarcodes)
-    const url = `${config.privateRuntimeConfig.labwhereBaseURL}/labwares?location_barcodes=${boxBarcodesParam}`
+    const url = `${config.privateRuntimeConfig.labwhereBaseURL}/labwares?location_barcodes=${boxBarcodes}`
```

Comment:
boxBarcodes is not a list unless I have got it wrong. I have used destructuring in the method header as in ...boxBarcodes which means it is passed as a number of arguments rather than an array. I just tried it to be sure and it seems to work. This means that you can pass a single barcode as well as an array and you don't have to modify anything.